src/Singleton/SingleTonClassImplementaion.java

```diff
@@ -0,0 +1,19 @@
+package Singleton;
+
+
+public class SingleTonClassImplementaion {
+    public static void main(String[] args)
+    {
+        Driver driver1= Driver.getInstance();
```

Comment:
Can u explore on how can one break Singleton and how u can restrict it

Reply:
Hi @mahidharrajujalumuru  i have covered single ton breakage and restrictions methods in this pr can u pls review it https://github.com/harichandhanabandaru/DesignPatterns/pull/4